pages/inspector/inspector-issues-beta.mdx

```diff
@@ -0,0 +1,61 @@
+# Inspector Issues View beta
+
+*Information and disclaimers for the Inspector issues view closed beta*
+
+## What is new in this beta?
+
+We've built a new infrastructure for generating issues, that allows for more flexibility and speed. That means that loading your issues will be faster and we're able to provide more detailed information in issue details. In addition we've added alerts for saved views.
+
+## How to get access to the beta
+
+Are you not in the beta, but are interested getting faster and better issues and alerts for saved views? Reach out to us via the chat bubble right here in the lower right corner or email us at beta@avo.sh!
+
+## What to look out for in this beta
+
+We are still tweaking the issue generation and the issue details so in the beginning you might encounter:
+- Issues with no issue event volume
+- Edge cases where an issue is displayed and alerted on that may not be an actual issue
+- We might wipe out all issues and regenerate them, which means an alert may be resent
```

Comment:
Don't remember anything else here, 
maybe a disclaimer that the issues will get regenerated with the same Id's so this won't spam their alerts